src/components/pages/Users/EditUserPanel/EditUserPanel.test.tsx

```diff
@@ -544,3 +554,98 @@ test("should handle errors when updating entitlements", async () => {
     }),
   ).toBeInTheDocument();
 });
+
+test("should change user details", async () => {
+  const invalidateQueries = vi.fn();
+  vi.spyOn(reactQuery, "useQueryClient").mockReturnValue({
+    invalidateQueries,
+  } as unknown as reactQuery.QueryClient);
+  let patchResponseBody: string | null = null;
+  let patchDone = false;
+  // eslint-disable-next-line @typescript-eslint/no-misused-promises
+  mockApiServer.events.on("request:start", async ({ request }) => {
+    const requestClone = request.clone();
+    if (
+      requestClone.method === "PUT" &&
+      requestClone.url.endsWith("/identities/user1")
+    ) {
+      patchResponseBody = await requestClone.text();
+      patchDone = true;
+    }
+  });
+  const {
+    result: { findNotificationByText },
+  } = renderComponent(
+    <EditUserPanel
+      user={mockUser}
+      onUserUpdate={() => void invalidateQueries({ queryKey: ["/identities"] })}
+      userId="user1"
+      close={vi.fn()}
+      setPanelWidth={vi.fn()}
+    />,
+  );
+  const firstNameField = screen.getByRole("textbox", {
+    name: UserPanelLabel.FIRST_NAME,
+  });
+  await userEvent.clear(firstNameField);
+  await userEvent.type(firstNameField, "First Name modified");
+  await userEvent.click(
+    screen.getByRole("button", { name: "Update local user" }),
+  );
+  await waitFor(() => expect(patchDone).toBe(true));
+  expect(patchResponseBody && JSON.parse(patchResponseBody)).toStrictEqual({
+    addedBy: "within",
+    email: "pfft@example.com",
+    firstName: "First Name modified",
+    id: "user1",
+    lastName: "good",
+    source: "noteworthy",
+  });
+  expect(
+    await findNotificationByText(
+      'User with email "pfft@example.com" was updated.',
+      {
+        appearance: "toast",
+        severity: NotificationSeverity.POSITIVE,
+      },
+    ),
+  ).toBeInTheDocument();
+  expect(invalidateQueries).toHaveBeenCalledWith({
+    queryKey: ["/identities"],
+  });
+});
+
+test("should handle errors when updating user details", async () => {
+  mockApiServer.use(getPutIdentitiesItemMockHandler400());
+  const invalidateQueries = vi.fn();
+  vi.spyOn(reactQuery, "useQueryClient").mockReturnValue({
+    invalidateQueries,
+  } as unknown as reactQuery.QueryClient);
+  const {
+    result: { findNotificationByText },
+  } = renderComponent(
+    <EditUserPanel
+      user={mockUser}
+      onUserUpdate={vi.fn()}
+      userId="user1"
+      close={vi.fn()}
+      setPanelWidth={vi.fn()}
+    />,
+  );
+  await userEvent.type(
+    screen.getByRole("textbox", { name: UserPanelLabel.FIRST_NAME }),
+    "First",
+  );
+  await userEvent.click(
+    screen.getByRole("button", { name: "Update local user" }),
+  );
+  expect(
+    await findNotificationByText(Label.USER_ERROR, {
+      appearance: "toast",
+      severity: NotificationSeverity.NEGATIVE,
+    }),
+  ).toBeInTheDocument();
+  expect(invalidateQueries).not.toHaveBeenCalledWith({
+    queryKey: ["/identities"],
+  });
```

Comment:
Because you've passed `onUserUpdate={vi.fn()}` I don't think this tests anything.